package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "fuego-photos",
+  "description": "Back end handler for user auth and photo uploading to S3",
+  "repository": "https://github.com/fuego-starter/fuego-photos.git",
+  "author": "",
+  "license": "Unlicense",
+  "private": true,
+  "scripts": {
```

Comment:
Great!  One thing I've done in the past is add one more script to allow a quick-start by combining the build-and-run into a single script.  Something along the lines of:
`"build:start": "./node_modules/.bin/tsc src/*.ts"`